docs/build-on-linea/tooling/cross-chain/shortcuts.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Onthis Shortcuts
+description: No-code tool for executing multiple DeFi actions in one transaction
+---
+
+A Shortcut allows users to execute onchain actions by sending funds to a contract address.
+
+Each Shortcut is a unique smart contract designed to carry out the desired function. Shortcuts are
+represented by ENS domains, making addresses human-readable.
+
+:::note
+
+Shortcuts currently works with Ethereum Mainnet only, unless explicitly specified by the Shortcut 
```

Comment:
...if they only work on Etherum Mainnet, why are we documenting it on Linea? I scrolled down to see if this is explained, but it's unclear